k8s/helmfile/env/production/platform-nginx.nginx.conf

```diff
@@ -243,6 +241,26 @@ server {
                 return 200 'gangnam style!';
         }
 
+        location = /version {
+                internal;
+                proxy_buffering on;
+                proxy_pass http://api-app-backend.default.svc.cluster.local/backend/ingress/getWikiVersionForDomain?domain=$host;
+                proxy_pass_request_body off;
+                proxy_set_header Content-Length "";
+                proxy_set_header X-Original-URI $request_uri;
+                proxy_cache version_cache;
+                proxy_cache_key $host;
+                proxy_cache_valid any 5m;
+                proxy_ignore_headers Cache-Control;
+                add_header X-Cache-Status $upstream_cache_status;
+	}
+
+        location = /version_error {
+                internal;
+                default_type text/html;
+                return 404 '<body>You have requested the domain: $host. But that wiki can not currently be loaded.<br/>It may never have existed or it might now be deleted.<br/>You can check the platform status at <a href="https://status.wikibase.cloud">status.wikibase.cloud</a>.</body>';
```

Comment:
the current error is also including `Failure looking up wiki asdfasd.wikibase.cloud.` (for example) but I don't think tweaking this a little bit matters